### PR TITLE
Changes main content width (Closes #367)

### LINKED ISSUE #367: Size of the main content
Width of 100% is too much. Maybe the use of 72% (960px/1320px) could be nice. The nav bar it's from the issue suggestion #366

size_content_1

### PR DESCRIPTION
**Pull request by DylanGuedes**
- Uses bootstrap container to enclose main content
- Adds dinamic stick footer. (mainly an enhancement for short pages)

### CODE REVIEW

File: app/views/shared/_footer.html.erb [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+  <div class="container">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-7">&copy; <%= link_to(t('mezuro_team'), '/humans.txt') %> 2013-2016</div>
+        <div class="col-md-5">
+          <div class="container-fluid">
+            <div class="row">
+              <div class="col-md-3"><%= link_to(image_tag('agplv3-88x31.png'), 'http://www.gnu.org/licenses/agpl-3.0-standalone.html') %></div>
+              <div class="col-md-3"><%= link_to image_tag('fork-me.png'), 'https://github.com/mezuro/prezento' %></div>
+              <div class="col-md-3"><%= image_tag 'usp-cloud-nuvem-logo.png' %></div>
```
Comment (rafamanzo) [inline app/views/shared/_footer.html.erb]: Maybe we should take this opportunity to remove this logo (we were kicked out of there).

File: app/views/shared/_footer.html.erb [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
-            <div class="col-md-6"><%= link_to image_tag('banner-ccsl.png', height: '40', width: '190'), 'http://ccsl.ime.usp.br/' %></div>
+<footer class="footer">
+  <div class="container">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-7">&copy; <%= link_to(t('mezuro_team'), '/humans.txt') %> 2013-2016</div>
+        <div class="col-md-5">
+          <div class="container-fluid">
+            <div class="row">
+              <div class="col-md-3"><%= link_to(image_tag('agplv3-88x31.png'), 'http://www.gnu.org/licenses/agpl-3.0-standalone.html') %></div>
```
Comment (rafamanzo) [inline app/views/shared/_footer.html.erb]: Why the cell width had to be changed?

**Comment by rafamanzo** [comment]
Are the footer changes related to modifying the main content width? They are indeed nice but keep in mind this comment from a previous PR: https://github.com/mezuro/prezento/pull/370#issuecomment-243573021

**Comment by rafamanzo** [comment]
Does the bootstrap `container` class provide the width limitation proposed by #367?

File: app/views/shared/_footer.html.erb (hunk 2 above)
Comment (DylanGuedes) [inline app/views/shared/_footer.html.erb]: Because the images are overlapping.

**Comment by DylanGuedes** [comment]
The footer changes are not related, but there's no issue for them and I thought that It was a nice addition - I can separate it in other merge request if you want.

About the width, I preferred to use the default bootstrap container (it isn't 72% I guess), because it works better for multiple resolutions (it's dinamic) than using 72% for every resolution (for a phone, for example, maybe a value higher than 72% looks better).

But I can use the fixed 72% width, if you think that it's better.

**Comment by rafamanzo** [comment]
If you rebase from master and remove that image from the footer I believe we are good to go.

**Comment by DylanGuedes** [comment]
Done.

**Comment by diegoamc** [comment]
Something odd is happening with the cucumber jobs for `ruby 2.3.0`. It seems `poltergeist` cannot be found. @DylanGuedes, is this something that happens on your local machine too?

**Comment by DylanGuedes** [comment]
In my local it looks ok, actually. :/
And I checked that the MRs opened by my friends are also broken for the same reason (something relative to capybara poltergeist missing).

**Pull request merged**